### PR TITLE
github/labeler: disable sync-labels

### PR DESCRIPTION
**Pull request by epuertat**
Yaml syntax cleaned too.

From https://github.com/wesnoth/wesnoth/commit/958c82d0867568057caaf58356502ec8c87d8366

Fixes: https://github.com/ceph/ceph/pull/38107#issuecomment-729300615
Signed-off-by: Ernesto Puerta <epuertat@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  triage:
+  labeler:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@main
-      with:
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+      - uses: actions/labeler@main
+        with:
+          sync-labels: ''
```
Comment (tchaikov) [inline .github/workflows/labeler.yml:10]: i think by default `sync-labels` is `false`. see https://github.com/actions/labeler/blob/fa244eaba5c3a196c5312fbef1e5ff0cf5350dc5/src/main.ts#L17 and https://github.com/actions/toolkit/blob/0759cdc2309acfd91b6503597dcfa02a3cee99d0/packages/core/src/core.ts#L84-L92

**Review by tchaikov — APPROVED** [review]

**Review by neha-ojha — APPROVED** [review]
Thanks @epuertat

**Review by djgalloway — APPROVED** [review]

File: .github/workflows/labeler.yml (hunk 1 above)
Comment (epuertat) [reply .github/workflows/labeler.yml:10]: @tchaikov after checking that code, my bet is that the defaults from the [`action.yml`](https://github.com/actions/labeler/blob/fa244eaba5c3a196c5312fbef1e5ff0cf5350dc5/action.yml) are somehow injected in the environment, and that's why the empty string doesn't work:
```s
> !!'false'
true

> !!''
false
```

Comment (epuertat) [reply .github/workflows/labeler.yml:10]: There it is: https://github.com/actions/toolkit/blob/1cc56db0ff126f4d65aeb83798852e02a2c180c3/docs/github-package.md#mocking-inputs

**Comment by epuertat** [comment]
I created this PR to see if we can get this fixed: https://github.com/actions/labeler/pull/113

**Pull request merged**